src/Workspaces/Core/Portable/Rename/RenameUtilities.cs

```diff
@@ -109,7 +109,8 @@ internal static IEnumerable<Document> GetDocumentsAffectedByRename(ISymbol symbo
         private static bool ShouldRenameOnlyAffectDeclaringProject(ISymbol symbol)
         {
             // Explicit interface implementations can cascade to other projects
-            return symbol.DeclaredAccessibility == Accessibility.Private && !symbol.ExplicitInterfaceImplementations().Any();
+            // check if the symbol is override. https://github.com/dotnet/roslyn/issues/25682
+            return symbol.DeclaredAccessibility == Accessibility.Private && !symbol.ExplicitInterfaceImplementations().Any() && !symbol.IsOverride;
```

Comment:
1. this comment is insufficient.
2. this expressoin is too complex to reason about.  please break up and comment appropriately.

Reply:
I don't understand how this fixes the referenced bug, or why it's a correct fix.

It seems like it is fine for this to return true.  if something else is asserting, we likely need to fix that assert.

Reply:
it's unclear ot me if this fix will actually fix whatever assert is firing.  Could you point me to the actual assert that is firing so i can tell?